Water leak Sensor Wiring through RJ11 Breakout Boards

BobGlicksman opened the pull request:
I have added a schematic of the wiring of a Water Leak Sensor through a Sparkfun RJ11 breakout board, through a 4 wire RJ11 telephone cable, to another Sparkfun RJ11 breakout and on to a 3 pin header on the water leak detector PCB/breadboard.  It works using > 10 feet of telephone cable between the Sparkfun boards.  Somehow, the RJ11 cable and/or the breakout boards crisscross some of the wires.  This schematic shows how it must be wired in order to work properly.

jschrempp committed:
Blynk is basically working.

jschrempp committed:
separated DHT read from particle publish so we can have different intervals for each

BobGlicksman committed:
Merge pull request #9 from jschrempp/withblynk

Adds Blynk functionality

jschrempp committed:
put ifdef's around the blynk code so it can be easily disabled

jschrempp committed:
Merge branch 'development' of https://github.com/BobGlicksman/WaterLeakSensor into withblynk

BobGlicksman committed:
Schematic of Sensor wiring with SF RJ11 PCB

The wiring of the Sparkfun RJ11 breakout boards through a 4 wire
telephne cable is screwy, to say the least!  Here is a schematic for how
it must be wired up.  Pins 3,4, and 5 are used on the sensor side, and
pins 2,3, and 4 are used on the Box side.  Sensor 5 --> Box 2 (signal),
Sensor 4 --> Box 3 (+3.3 volt) and Sensor 3 --> Box 4 (GND).  Tested
this with > 10 feet of telephone cable and it works fine.

BobGlicksman committed:
Merge pull request #10 from jschrempp/withblynk

Added a define at the top of the file to disable Blynk integration

jschrempp commented:
I wonder if the standard cables cross over txn and rcv.
    Jim
    
    
    On 3/27/17 11:55 AM, Bob Glicksman
      wrote:
    
    
      I have added a schematic of the wiring of a Water Leak Sensor
        through a Sparkfun RJ11 breakout board, through a 4 wire RJ11
        telephone cable, to another Sparkfun RJ11 breakout and on to a 3
        pin header on the water leak detector PCB/breadboard. It works
        using > 10 feet of telephone cable between the Sparkfun
        boards. Somehow, the RJ11 cable and/or the breakout boards
        crisscross some of the wires. This schematic shows how it must
        be wired in order to work properly.
      
      You can view, comment on, or merge this pull request online
        at:
        https://github.com/jschrempp/WaterLeakSensor/pull/5
      Commit Summary
      
        Blynk is basically working.
        separated DHT read from particle publish so we can have
          different intervals for each
        Merge pull request #9 from jschrempp/withblynk
        Schematic of Sensor wiring with SF RJ11 PCB
      
      File Changes
      
         M Firmware/WaterLeakDetector/project.properties
          (1) 
         M Firmware/WaterLeakDetector/src/WaterLeakDetector.ino
          (87) 
         A Hardware/RJ11
            sensor cabling.pdf (0) 
      
      Patch Links:
      
        https://github.com/jschrempp/WaterLeakSensor/pull/5.patch
        https://github.com/jschrempp/WaterLeakSensor/pull/5.diff
      
      —
        You are receiving this because you are subscribed to this
        thread.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/jschrempp/WaterLeakSensor","title":"jschrempp/WaterLeakSensor","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/jschrempp/WaterLeakSensor"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Water leak Sensor Wiring through RJ11 Breakout Boards (#5)"}],"action":{"name":"View Pull Request","url":"https://github.com/jschrempp/WaterLeakSensor/pull/5"}}}

BobGlicksman committed:
Final PCB Eagle Files

Final copies of Eagle schematic and board files as well as PDFs of
these.

BobGlicksman committed:
Moved PCB files to the correct folders for publication

BobGlicksman committed:
Corrected and renamed files

Changed Photon to through hole version.  Changed the layout somewhat to
improve placement and routing.  Changed the names to be more
descriptive.

BobGlicksman committed:
Updated parts list

Need to order parts and verify correct footprints for all (particularly
the piezzo buzzer).

BobGlicksman committed:
Updated parts list

BobGlicksman committed:
Added 4 pin SIPP socket to parts list

Added 4 pin SIPP socket to the parts list for mounting of the Adafruit
DHT11 on a PCB.

BobGlicksman committed:
Updated schematic

Updated schematic to show the resistor values for the components that
are on the parts list.
